README.md

```diff
@@ -1,14 +1,9 @@
-# Project
+# didx509go
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+[![go.dev](https://pkg.go.dev/badge/github.com/microsoft/didx509go.svg)](https://pkg.go.dev/github.com/microsoft/didx509go)
+[![tests](https://github.com/microsoft/didx509go/actions/workflows/ci.yml/badge.svg)](https://github.com/microsoft/didx509go/actions?query=workflow%3Aci)
```

Comment:
Just adding fancy badges on README.md.
[This](https://github.com/microsoft/didx509go/blob/d06c29af5ab9a61e14a1e950d115763b8df40b6a/README.md) is how it looks